packages/runtime/container-runtime/src/test/garbageCollection.spec.ts

```diff
@@ -114,6 +115,134 @@ describe("Garbage Collection Tests", () => {
         sessionStorageConfigProvider.value.getRawConfig = oldRawConfig;
     });
 
+    describe("Configuration", () => {
```

Comment:
Note: the `garbageCollection.spec.ts` is getting big.

Reply:
To match garbageCollection.ts ;)

Reply:
We definitely need these tests